### DISCUSSION TITLE
TuGraph Docker单机镜像环境中，如何获取每条cypher查询语句的耗时

### DISCUSSION THREAD

**Discussion by huzhihao1** [TuGraph-family/tugraph-db#408] [Q&A] [answered]
TuGraph Docker单机镜像环境中，如何获取每条cypher查询语句的耗时

**Comment by qishipengqsp** [comment, marked as answer]
使用cli调用cypher的时候可以看到时间
https://tugraph-db.readthedocs.io/zh-cn/latest/5.developer-manual/4.client-tools/5.tugraph-cli.html

**Reply by huzhihao1** [reply]
谢谢了
